### ISSUE TITLE
Paper 1.12.2: Issue Title

### ISSUE THREAD

**Issue by FrozenBeard** [kangarko/ChatControl-Pro#1866]
https://gist.github.com/FrozenBeard/7a3b21bc82d0528d7f0f47ddcc193032

**Comment by AngellitoX** [comment]
Hello, your issue log is unreachable. Can you please update it somewhere else?

**Comment by TomLewis** [comment]
I got approved for a username change on Github yesterday, looks like links don't update/forward.
I have updated the Gist with my new username (Previously FrozenBeard)

**Comment by AngellitoX** [comment]
Yup, it looks like there's something going on with SimpleClans. Could you please give steps to reproduce it?

Curiously, last night I started to use SimpleClans along with CCP, both latest versions and just checked my full logs and I haven't had any issues related. I have 150+ players online at the moment and the same thing, no errors.

Probably there's a problem with one of the placeholders.

**Comment by TomLewis** [comment]
To reproduce, people just send messages in global chat.

Maybe this is something to do with the placeholder? https://github.com/RoinujNosde/SimpleClans/wiki/PlaceholderAPI-Support

I am using these placeholders in my formatting.yml
```
{simpleclans_rank}
{simpleclans_clan_color_tag}
{simpleclans_in_clan}
{simpleclans_clan_balance}
{simpleclans_kdr}
{simpleclans_clan_total_kdr}
``` 

Please read the title in the GitHub ticket for versions.

**Comment by AngellitoX** [comment]
I am using these:
````
{simpleclans_tag_label}
{simpleclans_clan_name}
{simpleclans_rank_displayname}
````

And none of them are causing issues to me. You could probably the first 2 on your list for the first one on mine and run out some tests. Probably one of those placeholders are returning an improper value.

In order to know if the placeholder is fine, it would also be best to check if all of those placeholders are properly parsed into `/papi parse me <placeholder>` command.

**Comment by TomLewis** [comment]
**These placeholders are not new.** I have been using these for 5+ years with ChatControl Pro. Only after updating to 8.7.9 did they break. You tell me, is it the placeholders? Now I have given them to you, you can find out.

**Comment by TomLewis** [comment]
@kangarko Please can you come back to reading github bugs. You gave me direct answers/fixes instead of playing games!

**Comment by AngellitoX** [comment]
Ok, I'm going to stop responding to all of your tickets as for now. I was trying to help Matej while he's gone this week in the best of my abilities.

He will come back on Friday most likely, so I hope you can wait until then. Good luck!
